share/ansible/roles/ci_run/README.md

```diff
@@ -0,0 +1,25 @@
+Role Name
+=========
+
+Build, install and test package.
+
+Example Playbook
+----------------
+
+Usage example:
+
+    - hosts: builder
+      connection: podman
+      become: true
+      roles:
+        - role: ci_run
```

Comment:
I don't understand this usage.  Where is this YAML code supposed to be used?

Reply:
In the ansible playbook: https://github.com/shadow-maint/shadow/pull/1009/files#diff-1968d6dc5c6937169e73de5fa2e83bf0d9810afcec288aa45b6cfabb40a5bf15R12-R17

Roles are self-contained units in Ansible. They are used for grouping tasks and other resources in a known file structure.